addons/skin.estuary/xml/DialogAddonInfo.xml

```diff
@@ -245,6 +245,7 @@
 						<param name="id" value="12" />
 						<param name="icon" value="icons/infodialogs/launch.png" />
 						<param name="label" value="$LOCALIZE[518]" />
+						<param name="visible" value="!ListItem.Property(Addon.IsBinary)" />
```

Comment:
For now OK, but have in mind that also it can come that binary addon becomes runway to have independent. Maybe also some other addon types not need it.

There should be the most correct to have e.g. Addon.IsRunnable, to have not fixed to a language and where checks his type.

Reply:
Also not sure if them hidden that GUI his flow then match the others. Maybe only button disabled and not hidden 🤔 .

@Razzeee what you thing?

Reply:
Disabling would be better, but I might be missing some context here. I checked Leia and it seemed like we already did disable this button? Was that removed in between?

![image](https://user-images.githubusercontent.com/5943908/95675919-f78d7e80-0bba-11eb-93fc-a6fb91064eac.png)

Reply:
> Disabling would be better, but I might be missing some context here. Was that removed in between?

agreed and i would disable it in core, not the skin. think there hasn‘t been removed something in between but i also cannot „run“ `pvr.hts` for instance in leia